.babelrc

```diff
@@ -1,73 +1,71 @@
 {
-  "presets": [["@babel/preset-env", {
-    "useBuiltIns": "usage",
-    "modules": "auto",
-    "corejs": 3
-  }],
-  "@babel/preset-react",
-  "@babel/preset-typescript"],
-  "plugins": [
-    "@babel/plugin-proposal-optional-chaining",
-    "@babel/plugin-syntax-dynamic-import"
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": "usage",
+        "modules": "auto",
+        "corejs": 3
+      }
+    ],
+    "@babel/preset-react",
+    "@babel/preset-typescript"
   ],
   "env": {
-    "test": {
-      "plugins": ["@babel/plugin-transform-modules-commonjs"]
-    },
     "production": {
       "plugins": [
-        ["transform-imports", {
-          "@material-ui/core": {
-            "transform": "@material-ui/core/esm/${member}",
-            "preventFullImport": true
-          },
-          "@material-ui/icons": {
-            "transform": "@material-ui/icons/esm/${member}",
-            "preventFullImport": true
+        [
+          "babel-plugin-direct-import",
```

Comment:
did you check the bundle size without this plugin? according to their docs it should now tree shake out of the box and this plugin should only be necessary for older webpack versions or if you're worried about start time in development

Reply:
It doesn't look like it makes much difference in running dev instance asides from webpack taking 2x as long with the plugin 
![image](https://user-images.githubusercontent.com/25511396/148844368-b344c4d8-729a-4760-b211-d8a8ffe65f98.png)


However, orphan modules with the plugin is effectively cut in half in production build
![image](https://user-images.githubusercontent.com/25511396/148844423-25883b10-662e-4f8b-b3b2-7d990a5e8c0b.png)

Reply:
_with_ the plugin? I thought the plugin was supposed to make it faster. interesting about the orphan modules

Reply:
since this doesn't improve performance or affect bundle size, I think it would be better to ditch the plugin as it will make things simpler for future developers

Reply:
yeah, agreed, doesn't seem to add much